aiida/cmdline/commands/cmd_code.py

```diff
@@ -224,11 +224,8 @@ def show(code):
     table = []
     table.append(['PK', code.pk])
     table.append(['UUID', code.uuid])
-    table.append(['Label', code.label])
-    table.append(['Description', code.description])
-    table.append(['Default plugin', code.default_calc_job_plugin])
-    table.append(['Prepend text', code.prepend_text])
-    table.append(['Append text', code.append_text])
+    for key in code.get_cli_options().keys():
+        table.append([key.capitalize().replace('_', ' '), getattr(code, key)])
```

Comment:
We need to catch the `AttributeError` that will be thrown in case no property exists for the cli option. This is the case for the `filepath_files` option for the `PortableCode` class. This is used in the CLI to point to the path containing the files of the code, but these are then stored in the repository. For a stored version of the code then, the `filepath_files` loses any meaning, which is why there isn't a corresponding property.
```suggestion
        try:
            table.append([key.capitalize().replace('_', ' '), getattr(code, key)])
        except AttributeError:
            continue
```
If we think what we would _want_ it to show, is probably a list (or tree view) of the repository contents, but don't think this is necessary for this PR. So let's just catch the exception and skip it.

Reply:
Also, maybe we add the `table.append(['Type', code.entry_point.name])` entry after the `UUID`. This value is also displayed by `verdi code list` and helps determine the type of code now that we have different subclasses.

Reply:
Thanks for detailed pointers @sphuber . I've addressed these.